internal/relay/pinpoint/relay.go

```diff
@@ -0,0 +1,75 @@
+package relay
+
+import (
+	"encoding/json"
+	"fmt"
+	"net"
+	"os"
+	"time"
+
+	"github.com/aws/aws-sdk-go/service/pinpointemail"
+	"github.com/aws/aws-sdk-go/service/pinpointemail/pinpointemailiface"
+)
+
+// Request object
+type Request struct {
+	Time  time.Time
+	addr  net.Addr
+	IP    string
+	From  string
+	To    []string
+	Error string
+	err   *error
+}
```

Comment:
Since this is exactly the same struct as in the SES Relay, this should be in a shared library.

Reply:
👍 Done.